packages/fetchye/__tests__/FetchyeProvider.spec.jsx

```diff
@@ -16,7 +16,7 @@
 
 import React, { useContext, useEffect } from 'react';
 import { render } from '@testing-library/react';
-import * as actions from '../src/cache/actions';
```

Comment:
can we leave this scoped under cache

Reply:
@JAdshead I think we moved the `simpleCache` out of the cache folder as well, we are not expecting to add anymore cache providers inside the fetchye package but I guess what you are suggesting is to keep it in the cache folder so it is more organised?